apps/ucp-ui/src/Layout/SideNav.tsx

```diff
@@ -28,6 +36,11 @@ const SideNav = () => {
   const { pathname } = useLocation();
 
   const links = [
+    {
+      label: SIDE_NAV_INSTITUTIONS_LINK_TEXT,
+      Icon: AccountBalanceOutlined,
```

Comment:
Does `Icon` need to be a different case from the other properties here?

Reply:
Yeah it does. React won't let you render anything unless it starts with a capital letter